templates/profile.mustache

```diff
@@ -161,6 +161,10 @@
                     <th>{{#str}} field_dbreplicareads, tool_excimer {{/str}}</th>
                     <td>{{dbreplicareads}}</td>
                 </tr>
+                <tr>
+                    <th>Lock reason</th>
```

Comment:
1) this should nbe a lang string

2) can you move this out of the table and put it under just as text. Can you run it through the output api to turn any urls into links:

![image](https://user-images.githubusercontent.com/187449/189516128-c9f97635-b40a-4ed7-bb1e-40bd9da29de4.png)

Reply:
Actually can you put this at the top, and put it inside a notification box saying 'This is locked' with the reason text under that, and then the table